docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -4,7 +4,7 @@ To modify the behavior of existing blocks, WordPress exposes several APIs:
 
 ### Block Style Variations
 
-Block Style Variations allow providing alternative styles to existing blocks. They work by adding a className to the block's wrapper. This className can be used to provide an alternative styling for the block if the style variation is selected.
+Block Style Variations allow providing alternative styles to existing blocks. They work by adding a className to the block's wrapper. This className can be used to provide an alternative styling for the block if the style variation is selected. See the [Getting Started with JavaScript tutorial](../../../../docs/designers-developers/developers/tutorials/javascript/) for a full example.
```

Comment:
I think @dd32 can confirm but we should be able to do a simpler URL here:

```
../../../../docs/designers-developers/developers/tutorials/javascript/
```

can become:

```
/docs/designers-developers/developers/tutorials/javascript/
```

(This has always worked on GitHub but I believe our Markdown parser will catch it too now.)

Reply:
That would be great! Switched and can try it out when published :+1: